scripts/run.sh

```diff
@@ -33,7 +33,7 @@ if [ ! -d "distributed" ]; then
 fi
 
 pip uninstall dask distributed
-cd dask && git clean -fdx && git checkout $DASK_VERSION && pip install -e .[test] && cd ..
-cd distributed && git clean -fdx && git checkout $DASK_VERSION && pip install -e . && cd ..
+cd dask &&  pip install -e .[test] && cd ..
+cd distributed && pip install -e . && cd ..
```

Comment:
Do we not install `rapids-dask-dependency` to run tests here? If we did then this choice would be automatic.

Reply:
IIUC, rapids-dask-dependency will eventually depend on `dask>=2025.1.0` or something like that. When we do that, I think we'll still want to run the tests with dask main (rather than a released version), to ensure that no changes dask / distributed on main have caused regressions.

Reply:
Ah ok, I thought this was just "unpinning" to test it now like we did in rapids-dask-dependency. I guess technically this is then fixing a bug here, as it was always the intent to test against main, is that right?

Reply:
Right, forgot to mention that. Previously it was crashing on import with dask `main`, so we pinned the tests here to get something running.